src/test/regress/Makefile

```diff
@@ -64,12 +65,14 @@ check-multi-task-tracker-extra: all tempinstall-main
 	$(pg_regress_multi_check) --load-extension=citus \
 	--server-option=citus.task_executor_type=task-tracker \
 	--server-option=citus.large_table_shard_count=1 \
+	--server-option=citus.shard_replication_factor=2 \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/multi_task_tracker_extra_schedule $(EXTRA_TESTS)
 
 
 check-multi-binary: all tempinstall-main
 	$(pg_regress_multi_check) --load-extension=citus \
 	--server-option=citus.binary_worker_copy_format=on \
+	--server-option=citus.shard_replication_factor=2 \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/multi_binary_schedule $(EXTRA_TESTS)
```

Comment:
Hm, I think I'd rather set it in pg_regress_multi than specify it in multiple tests here?